jobs/jobs_watcher.go

```diff
@@ -12,21 +12,23 @@ import (
 )
 
 const (
-	WATCHER_POLLING_INTERVAL = 15000
+	DEFAULT_POLLING_INTERVAL = 15000
```

Comment:
Can you call this `DEFAULT_WATCHER_POLLING_INTERVAL` as I don't want it to get confused with the other polling intervals that exist in the `jobs` package?

Reply:
Yup